server/schema/comment/comment.mutation.js

```diff
@@ -0,0 +1,69 @@
+const {
+  GraphQLObjectType,
+  // GraphQLScalarType,
+  // GraphQLUnionType,
+  // GraphQLInputObjectType,
+  // GraphQLEnumType,
+  // GraphQLInterfaceType,
+  // GraphQLSchema,
+  GraphQLNonNull,
+  // GraphQLError,
+  GraphQLList,
+  GraphQLString,
+  GraphQLID,
+  // GraphQLBoolean,
+  // GraphQLInt,
+  // GraphQLFloat,
+  // GraphQLDate,
+  // GraphQLTime,
+  // GraphQLDateTime,
+  // GraphQLJSON,
+  GraphQLJSONObject,
+} = require('../scalars');
+const { createComment, updateCommentAuthor, deleteComment } = require('./comment.resolver');
+
+const CommentType = require('./comment.type');
+
+module.exports = new GraphQLObjectType({
+  name: 'CommentMutation',
+  fields: {
+    createComment: {
+      description: 'Create a comment',
+      type: CommentType,
+      args: {
+        content: { type: new GraphQLNonNull(new GraphQLList(GraphQLJSONObject)) },
```

Comment:
@Shurtu-gal why the content type is like this, why not simply a string?

Reply:
Will make it a string. This was for styling in the comment.